pokemongo_bot/cell_workers/complete_tutorial.py

```diff
@@ -1,214 +0,0 @@
-import random
```

Comment:
So should I keep the files in that folder? Or a new one, since it works a little different.